.gitignore

```
@@ -19,6 +19,7 @@ target/
 # files
 **/*.so
 **/*.sqlite
+**/*.sqlite-jounal
```

jounal or journal?

Anyhow:

```suggestion
**/*.sqlite*
```

?

---

Yes, this :upside_down_face: